app/api/user/service.py

```diff
@@ -10,3 +11,17 @@ def get_user_by_id(db: Session, user_id: UUID):
 def delete_user_by_id(db: Session, user_id: UUID):
     db.query(models.User).filter(models.User.id == user_id).delete()
     db.commit()
+
+
+def update_user_by_id(db: Session, user_id: UUID, user_update: schemas.User):
```

Comment:
```suggestion
def update_user(db: Session, user_id: UUID, payload: UserUpdate):
```

原本的命名意思是透過 `user_id` 更新使用者，但函式實際不是做這樣的事情